Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANS/SANSSave.py

```diff
@@ -127,7 +127,7 @@ def _get_file_formats(self):
 
         # SaveNXcanSAS clashes with SaveNexusProcessed
         self.add_file_format_with_appended_name_requirement(file_formats, SaveType.NXcanSAS, file_types,
-                                                            [SaveType.Nexus])
```

Comment:
I don't understand this change, or what it did before.

Reply:
What this did was add an optional suffix to the end of the file name when you were saving both a Nexus processed file and a NXcanSAS file as otherwise they would both have the same name and extension and so overwrite each other. Now that NXcanSAS has a h5 extension this is no longer needed.